src/components/ItemSet/index.js

```diff
@@ -1,13 +1,23 @@
+import * as ls from 'app/lib/ls';
```

Comment:
could you move this down to above `import Item from 'app/components/Item';`? I keep all external dependencies first, then our app-level imports.